tasks/clientlibs.js

```diff
@@ -37,6 +37,7 @@ module.exports = function (grunt) {
 			},
 			config = {
 				clientLibPath: './clientlibs/',
+				compressCSS: true,
```

Comment:
If you set default value as un-compress CSS, then, we don't need to change AEM code ~_~